### PR TITLE
[4253][15.0][MIG] stock_quant_list_view: Migration to 15.0

### PR DESCRIPTION
**Pull request by AungKoKoLin1997**
[4253](https://www.quartile.co/web#cids=3&menu_id=505&action=1457&model=project.task&view_type=form&id=4253)

### CODE REVIEW

**Review by yostashiro — COMMENTED** [review]
@kanda999 Can you double-check if we need to keep this module?

File: stock_quant_list_view/views/stock_quant_views.xml [hunk 1]
```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+    <record model="ir.ui.view" id="view_stock_quant_tree">
+        <field name="name">stock.quant.tree</field>
+        <field name="model">stock.quant</field>
+        <field eval="10" name="priority" />
+        <field name="arch" type="xml">
+            <tree create="0" delete="0" decoration-danger="quantity &lt; 0">
+                <field name="product_id" />
```
Comment (kanda999) [inline stock_quant_list_view/views/stock_quant_views.xml]:
```suggestion
                <field name="product_id" optional="show" />
```
Can you please add optional?

**Review by kanda999 — CHANGES_REQUESTED** [review]
@AungKoKoLin1997 
The customer needed this view and decided to leave this moudle.

**Review by kanda999 — APPROVED** [review]
Code and Functional review

**Pull request merged**